typescript/conditions.d.ts

```diff
@@ -23,7 +23,7 @@ export type ConditionalPredicateNumeric = ConditionalPredicateBase & {
 
 export type ConditionalPredicateOptions = ConditionalPredicateBase & {
   /** Evaluate against a numeric type element or an element with options */
-  type: 'OPTIONS'
+  type?: 'OPTIONS'
```

Comment:
why would this be optional?

Reply:
`type` property did not exist (long time ago) there are predicates without a `type` property. `ConditionalPredicateOptions` has been wrong for a long time.